.github/workflows/pr-command.yml

```diff
@@ -1,31 +1,36 @@
-name: CI build and push (PR)
+name: PR Command
 
 concurrency:
-  group: pr-${{ github.event.number }}
+  group: ci-${{ github.run_id }}
   cancel-in-progress: true
 
 on:
-  pull_request_target:
-    types:
-      - opened
-      - reopened
-      - synchronize
-      - labeled
-      - unlabeled
-    branches:
-      - main
-      - v[0-9]+
-      - v[0-9]+.[0-9]+
-      - cryostat-v[0-9]+.[0-9]+
+  issue_comment:
+    types: [created]
 
 jobs:
+  create-command:
```

Comment:
Wondering what purpose of this job? 

I think the ` xt0rted/slash-command-action@v1` action should be used as the first step in other jobs that run on on PR command. Basically, an additional check if the command is supported (i.e other check is with the `if` clause).

Reply:
I think we can just not use this. Instead, use `if` clause only + an action to add a reaction to the PR comment. That should do the exact same thing.

Reply:
how do I add a reaction to a PR comment?